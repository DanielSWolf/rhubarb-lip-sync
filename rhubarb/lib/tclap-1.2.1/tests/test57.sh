#!/bin/sh

# failure
# This used to fail on the "Too many arguments!" but now fails sooner,
# and on a more approriate error.
../examples/test5 --aaa asdf -c fdas --fff blah -i one -i two -j huh > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test57.out; then
	exit 0
else
	exit 1
fi

