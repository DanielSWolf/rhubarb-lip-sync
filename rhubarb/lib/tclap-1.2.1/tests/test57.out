PARSE ERROR: Argument: -i (--iii)
             Mutually exclusive argument already set!

Brief USAGE: 
   ../examples/test5  {-a <string>|-b <string>} {--eee <string>|--fff
                      <string>|-g <string>} {-i <string> ... |-j <string>
                      ... } [--ddd] -c <string> [--] [--version] [-h]

For complete USAGE and HELP type: 
   ../examples/test5 --help

